distributed/utils_test.py

```diff
@@ -875,7 +875,7 @@ def gen_cluster(
     active_rpc_timeout: float = 1,
     config: dict[str, Any] | None = None,
     clean_kwargs: dict[str, Any] | None = None,
-    allow_unclosed: bool = False,
+    allow_unclosed: bool = True,
```

Comment:
I'd suggest adding a `FIXME` and linking to #8054 or a dedicated issue for leaking comms (which would need to get created).

```suggestion
    # FIXME: distributed#8054
    allow_unclosed: bool = True,
```